tests/lava/magma/runtime/test_ref_var_ports.py

```diff
@@ -0,0 +1,239 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+import numpy as np
+import unittest
+
+from lava.magma.core.decorator import implements, requires
+from lava.magma.core.model.py.model import PyLoihiProcessModel
+from lava.magma.core.model.py.ports import PyRefPort, PyVarPort
+from lava.magma.core.model.py.type import LavaPyType
+from lava.magma.core.process.ports.ports import RefPort, VarPort
+from lava.magma.core.process.process import AbstractProcess
+from lava.magma.core.process.variable import Var
+from lava.magma.core.resources import CPU
+from lava.magma.core.sync.domain import SyncDomain
+from lava.magma.core.sync.protocols.loihi_protocol import LoihiProtocol
+from lava.magma.core.run_configs import RunConfig
+from lava.magma.core.run_conditions import RunSteps
+
+
+# minimal process with an OutPort
+class P1(AbstractProcess):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self.ref = RefPort(shape=(3,))
+        self.v = Var(shape=(2,), init=17)
+        self.var_port_read = VarPort(self.v)
```

Comment:
Can we maybe have better distinctive names for ports and vars to distinguish reading and writing scenario? Because it confused me and took me some time to figure it out

Reply:
The scenario is only clear, when the connections are made.  They are used for different scenarios.
I renamed the ports and Vars to be more recognizable.